manifests/bind_exporter.pp

```diff
@@ -0,0 +1,138 @@
+# Class: prometheus::bind_exporter
```

Comment:
@anarcat If you introduce a new exporter, you should also provide the files ```spec/acceptance/bind_exporter.rb```,  ```spec/classes/bind_exporter.rb```. Otherwise the new exporter will not be tested.

Reply:
@dhoppe totally agreed!